component-resolver-core/src/main/groovy/org.octopusden/octopus/escrow/configuration/loader/EscrowConfigurationLoader.groovy

```diff
@@ -937,6 +952,13 @@ class EscrowConfigurationLoader {
         return new Distribution(explicit, external, GAV, DEB, RPM, docker, securityGroup)
     }
 
+    @TypeChecked(TypeCheckingMode.SKIP)
+    static final Doc parseDocSection(ConfigObject docConfigObject) {
+        def component = docConfigObject.containsKey("component") ? docConfigObject.component : null
```

Comment:
component is not nullable in Doc section so null is not allowed. 
In production the server loads config via loadFullConfigurationWithoutValidationForUnknownAttributes so the “Doc.component is not specified…” error path is bypassed.

Reply:
Fixed